canonical/lxd-ui
# Use upstreamed notifications [WD-5066]

## Done

- Replaced local hook and components with those upstreamed to react-components (`NotificationProvider` and `NotificationConsumer`)

Fixes [WD-5066](https://warthogs.atlassian.net/browse/WD-5066)

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @lorumic or @edlerd for access.
    - With a local copy of this branch, run as described [here](../HACKING.md#setting-up-for-development).
2. Perform the following QA steps:
    - Check that app-wide notifications that use the `useNotify` hook are working as expected

[WD-5066]: https://warthogs.atlassian.net/browse/WD-5066?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
